hub/core/chunk_engine/tests/common.py

```diff
@@ -142,8 +142,8 @@ def run_engine_test(
                 "chunk_size": chunk_size,
                 "length": a_in.shape[0],
                 "dtype": a_in.dtype.name,
-                "min_shape": tuple(a_in.shape[1:]),
-                "max_shape": tuple(a_in.shape[1:]),
+                "min_shape": list(a_in.shape[1:]),
```

Comment:
I think tuple is probably better here. No need for mutability, right?

Reply:
There is no need for mutability, however since we are moving from pickle->json. We will have to store the shapes as lists as json doesn't support tuples.